proposed/http-middleware/middleware.md

```diff
@@ -73,34 +72,34 @@ interface MiddlewareInterface
 {
     /**
      * Process an incoming server request and return a response, optionally delegating
-     * to the next middleware component to create the response.
+     * response creation to a handler.
      *
      * @param ServerRequestInterface $request
-     * @param DelegateInterface $delegate
+     * @param HandlerInterface $handler
      *
      * @return ResponseInterface
      */
     public function process(
         ServerRequestInterface $request,
-        DelegateInterface $delegate
+        HandlerInterface $handler
```

Comment:
I'm agreeing with @shadowhand here; `$next` implies that the handler has some sort of stack/queue behind it. `$delegate` would work, but would raise questions about why the typehint is `HandlerInterface` (or the counter-proposal I did of `RequestHandlerInterface`/`ServerRequestHandlerInterface`). Having it as `$handler`, with clear text in the proposal as to its purpose (_to handle a request and produce a response if the middleware cannot produce one on its own_) should be sufficient.